test/qa/test_connect.py

```diff
@@ -469,4 +470,69 @@ def test_connect_to_unavailable_servers(tech, proto, obfuscated):
             sh.nordvpn.connect(name)
 
         print(ex.value)
-        assert lib.is_connect_unsuccessful(ex)
\ No newline at end of file
+        assert lib.is_connect_unsuccessful(ex)
+
+
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_status_disconnected(tech, proto, obfuscated):
```

Comment:
Maybe this could be merged with `test_status_connected`? Just put the assertion after technology and protocol setup, in line 491.

Reply:
Good point, changed, thanks!